build/static/style.css

```diff
@@ -36,9 +36,18 @@ td {
 }
 
 .table-legend {
-  margin-left: 10%;
   max-width: 50%;
 }
 .table-header-label {
   text-align: center;
+}
+#report-explanation {
```

Comment:
the usage of id's is pretty wide i this document. I'd opt for classNames since nothing here really needs to be 'unique'